api/src/main/resources/migrations/surgicalAppointmentAttributeTypes.xml

```diff
@@ -41,12 +43,13 @@
         </preConditions>
         <comment>Adding estTimeMinutes attribute into surgical_appointment_attribute_type</comment>
         <sql>
+            SELECT user_id into @userid from users where username = 'admin';
```

Comment:
Same comments as above for the 'splitStatements' property on changeset if using multiple SQL statements.